config/environments/test.rb

```diff
@@ -43,6 +43,9 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Default mailer URL for test env only!
+  config.action_mailer.default_url_options = { host: "www.example.com" }
```

Comment:
Possibly overkill for this specific bugfix, but could come in handy for improving the mailer tests in future

Reply:
Good idea